src/libs/bir_expLib.sml

```diff
@@ -1,92 +1,66 @@
 structure bir_expLib =
 struct
 
+  local
```

Comment:
what is the advantage of this local?

Reply:
That's the pattern I'm using in all the files:
```
structure someLib =
struct

  local

  open HolKernel Parse boolLib bossLib;
  more opens;

  val ERR = Feedback.mk_HOL_ERR "someLib";
  val wrap_exn = Feedback.wrap_exn "someLib";

  in

  exported stuff

  end (* local *)

end (* someLib *)
```
Like this, you don't export too much stuff. (do you really want to export `ERR`?)

Reply:
good point. so in this way you manage this ad-hoc interface creation. in the end of the day, we should get used to a cleaner practice and create interfaces I guess, but you're absolutely right in this. good that I asked :)